kie-maven-plugin/pom.xml

```diff
@@ -213,15 +223,19 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>io.takari.maven.plugins</groupId>
-      <artifactId>takari-plugin-integration-testing</artifactId>
-      <type>pom</type>
+      <groupId>junit</groupId>
```

Comment:
Declaring junit dep is redundat because all descendants of kie-parent get it from here https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L2326-L2330